pages/productListingPage/productListingPage.css

```diff
@@ -0,0 +1,18 @@
+@import "/styles.css";
+
+:root {}
+
+body {}
+
+/* main content */
+.main-content {
+    width: 100%;
+    height: 100%;
+    display: grid;
```

Comment:
put width:100% and height: 100% in single class and use

Reply:
.main-content  will always be 100% x 100%

Above statement is specific to the project structure that I am following. Following is being done to decouple grid section with the main "main" section if decoupling was not done then that 100% x 100% would not have been required.